From 57f548a2a4b9e279bec18d03728e42c2baa7e5be Mon Sep 17 00:00:00 2001
From: Chuanxiao Dong <chuanxiao.dong@intel.com>
Date: Wed, 30 Jul 2014 13:25:11 +0800
Subject: [PATCH 12/15] mmc:host: change eMMC HS200 encoding in UHS Mode

On all platform SDHC host controller implementation,
it encoding eMMC HS200 (SDR200) the same value '3' as
SDR104 in Host controller 2 Register UHS Mode Select
field. But the Android storage driver just define
that value as '5', which means 'Reserved' according
to controller Spec.

This may cause potential issues. To prevent that,
submit this patch to change the value define from
'5' to '3'.

Change-Id: Ied743af7a9d4aae1bbc61d380a60e5d9ab97b653
Signed-off-by: yisun1 <yi.y.sun@intel.com>
Signed-off-by: Chuanxiao Dong <chuanxiao.dong@intel.com>
---
 drivers/mmc/host/sdhci.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/host/sdhci.h b/drivers/mmc/host/sdhci.h
index 4dfabed6d3ee..dc9f2be552cc 100644
--- a/drivers/mmc/host/sdhci.h
+++ b/drivers/mmc/host/sdhci.h
@@ -161,7 +161,7 @@
 #define   SDHCI_CTRL_UHS_SDR50		0x0002
 #define   SDHCI_CTRL_UHS_SDR104		0x0003
 #define   SDHCI_CTRL_UHS_DDR50		0x0004
-#define   SDHCI_CTRL_HS_SDR200		0x0005 /* reserved value in SDIO spec */
+#define   SDHCI_CTRL_HS_SDR200		SDHCI_CTRL_UHS_SDR104
 #define  SDHCI_CTRL_VDD_180		0x0008
 #define  SDHCI_CTRL_DRV_TYPE_MASK	0x0030
 #define   SDHCI_CTRL_DRV_TYPE_B		0x0000
-- 
1.9.1

